mobile/examples/super_resolution/android/app/src/main/java/ai/onnxruntime/example/superresolution/MainActivity.kt

```diff
@@ -30,8 +30,11 @@ class MainActivity : AppCompatActivity() {
         outputImage = findViewById(R.id.imageView2);
         superResolutionButton = findViewById(R.id.super_resolution_button)
 
+        val sessionOptions: OrtSession.SessionOptions = OrtSession.SessionOptions()
+        sessionOptions.registerCustomOpLibrary(OrtxPackage.getLibraryPath())
+
```

Comment:
As this is an example we should have comments explaining why we're doing things as the reader is unlikely to be familiar with ORT and this is how we can teach them what to do.

e.g. 'Register the onnxruntime extensions package that contains the custom operators. These are used to decode the input image into the format the original model requires, and to encode the model output into png format'

Can we also use the new 'registerCustomOpsUsingFunction' function so the examples are consistent across platforms?

Reply:
Make sense. thanks for the suggestion. will add comments and notes here.

> Can we also use the new 'registerCustomOpsUsingFunction' function so the examples are consistent across platforms?

Regarding this, yes we should definitely update to this method for registering the custom ops. 

haven't get it tested yet (for iOS as well.) will update.